src/hooks/use-is-fa2-token.hook.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+
+import { useSelectedAccountSelector } from '../store/wallet/wallet-selectors';
+import { useReadOnlyTezosToolkit } from './use-read-only-tezos-toolkit.hook';
+
+export const useIsFA2Token = (address: string) => {
+  const selectedAccount = useSelectedAccountSelector();
+  const tezos = useReadOnlyTezosToolkit(selectedAccount);
+
+  const [loading, setLoading] = useState(false);
+  const [isFa2, setIsFa2] = useState(false);
+
+  useEffect(() => {
+    async function getTokenStandart() {
+      try {
+        setLoading(true);
+        const contract = await tezos.contract.at(address);
+        const isTokenFa2 = contract.methods.update_operators;
+        setIsFa2(Boolean(isTokenFa2));
```

Comment:
try to reuse `getTokenType` as it implements a more complex token standard assertion.  
plus, refactor this hook as `useTokenType` hook, this will be more useful in the future.